README.md

```diff
@@ -1,2 +1,118 @@
-# zulip-gci-submissions-2017
-Student submissions for Zulip's GCI 2017 program
+# Zulip Google Code-In Submissions Repository
+
+Welcome to the Zulip GCI landing page!
+
+This page has information on [joining the developers' chat server](#community)
+and [setting up a development
+environment](#setting-up-the-zulip-development-environment).
+
+Once you're done, we recommend browsing through Zulip's [extensive
+documentation](http://zulip.readthedocs.io/en/latest/readme-symlink.html) about
+its code and community. If you're planning on submitting a lot of code, we
+particularly recommend the guides to
+[git](https://zulip.readthedocs.io/en/latest/git-guide.html), [version
+control](https://zulip.readthedocs.io/en/latest/version-control.html), [coding
+style](https://zulip.readthedocs.io/en/latest/code-style.html), and
+[testing](https://zulip.readthedocs.io/en/latest/testing.html).
```

Comment:
I think we actually want the README.md content to stay in the zulip-gci repository; this should just have a brief version that links over to that.  My reasoning is that our strategy is to have all the actual documentation content in zulip-gci.git, with this repository just containing the year's student submissions.

So I think you should submit this README.md changeset as an update to the existing zulip-gci repo?